recipes/dashing/meta.yaml

```diff
@@ -11,21 +11,21 @@ build:
 source:
   git_url: https://github.com/dnbaker/dashing
   git_rev: v{{ version }}
-  sha256: {{sha256}}
+  # sha256: {{sha256}} How is this found?
```

Comment:
```suggestion
  sha256: unused
```
I have not used `git_url` for packages in the past, but it seems like it is [what most other members do](https://github.com/bioconda/bioconda-recipes/search?q=git_url&unscoped_q=git_url). You may also need to add `- missing_hash` to your `skip-lints:` section.